### ISSUE TITLE
The color contrast on your site at https://gatsbyconf.com/ is not accessible

### ISSUE THREAD

**Issue by cjwest** [gatsbyjs/gatsby#34824]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

The color contrast on the homepage: https://gatsbyconf.com/ does not meet contrast guidelines.

### Reproduction Link

https://gatsbyconf.com/

### Steps to Reproduce

1. Navigate to your home page
2. run a color contrast test
3. See that the color contrast doesn't meet accessibility standards
...

### Expected Result

From your accessibility statement
> We strive to make these websites and the Gatsby framework itself as accessible as possible. Our goal is to meet [WCAG 2.1 AA](https://www.w3.org/TR/WCAG21/), with which we are partially compliant. We’ll be the first to admit the Gatsby ecosystem is a work in progress, and we are open to all feedback to make things better.

To contact the core team with your accessibility feedback or challenges, please [file an issue on GitHub](https://github.com/gatsbyjs/gatsby/issues/new/choose).

Your site should meet contrast standards

### Actual Result

![Screen Shot 2022-02-15 at 1 32 07 PM](https://user-images.githubusercontent.com/284440/154153506-e4750dcf-e9e0-4914-9901-55b5ef1268e5.png)

![Screen Shot 2022-02-15 at 1 39 46 PM](https://user-images.githubusercontent.com/284440/154153858-18fcc54a-6342-4161-841f-94479fe16ee7.png)

### Environment

```shell
n/a
```

**Comment by LekoArts** [comment]
Hi, thanks for the issue! I'll forward it internally.
